oeo.omn

```diff
@@ -3414,7 +3456,8 @@ Class: NonRenewableMunicipalWaste
         <http://purl.obolibrary.org/obo/IAO_0000119> "REGULATION (EC) No 1099/2008 OF THE EUROPEAN PARLIAMENT AND OF THE COUNCIL of 22 October 2008 on energy statistics https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX:32008R1099&from=EN"@en
     
     SubClassOf: 
-        MunicipalWaste
+        MunicipalWaste,
+        has_origin value fossil
```

Comment:
here origin is also biogenic and renewable?

Reply:
I intentionally applied no `has_origin` to `MunicipalWaste` as municipal waste per se has no _definied_ origin. Municipal waste has both an fossil and a biogenic/renewable fraction.

If we add `has_origin value fossil`, `has_origin value biogenic` and `has_origin value renewable` then this implies that municipal waste is fossil and biogenic/renewable at the same time.

If we do not add any `has_origin` to municipal waste than this implies that municipal waste has no defined origin. I wanted to express here, that only the subclasses `RenewableMunicipalWaste`and `NonRenewableMunicipalWaste` have defined origins. So not adding a `has_origin` to the classe `MunicipalWaste` is here in my view not a bug but a feature.